lib/csv.rb

```diff
@@ -936,7 +943,9 @@ def initialize(data,
                  write_converters: nil,
                  write_nil_value: nil,
                  write_empty_value: "",
-                 strip: false)
+                 strip: false,
+                 undef: nil,
+                 replace: nil)
```

Comment:
Should we need to accept `:undef` and `:replace` in `CVS#initialize`?
It seems that we just need to accept them in `CSV.open`. Then we can remove them from `options` in `CSV.open` arguments.

Reply:
I will fix  the arguments in CSV.open.